File: WAA/lab3/src/main/java/edu/miutest/demo/entity/Post.java
package edu.miutest.demo.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Entity
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    long id;
    String title;
    String content;
    String author;
    @OneToMany
    @JoinColumn(name ="comment_id")
    List<Comment> comments;

}

File: WAA/react/demo/src/components/Post.js
import React from "react";

const Post = ({post}) => {
    return (
        <div className="Content">
            
            <div className="Field" >

                <h3> Id: {post.id}</h3>
                <h3> Title: {post.title}</h3>
                <h3> Author: {post.author}</h3>

            </div>
        </div>
    );
}

export default Post